source/hardware/controllers/denon_mc7000.rst

```diff
@@ -308,7 +308,7 @@ LEDs
 
 The Channel Volume Meters matches to the ones shown in Mixxx GUI. Only when clipping the red LED illuminates.
 
-The Master Volume Meter is not correlated to Mixxx GUI as the controller handles that in Hardware.
+The Main Volume Meter is not correlated to Mixxx GUI as the controller handles that in Hardware.
```

Comment:
I'm unsure whether we should change that since "Main Volume Meter" seems to be referring to the meter which is in the region labelled "master" on the controller. 
![image](https://user-images.githubusercontent.com/12380386/127520790-45939fd3-1fff-4548-9c3a-0f00524722f5.png)

Reply:
That depends on the device.

![](https://theaverageguy.tv/wp-content/uploads/2013/11/MultiMix4USB-xlarge.jpg)

![](https://i5.walmartimages.com/asr/550f8c16-fbc1-42f1-b465-95a1a98f8535_1.ec77902d05d925eafa1c5dcebabd0ad8.jpeg)

Reply:
I fully agree that its device dependent (I was actually positively surprised seeing how much hardware was already using main instead of master). However, the screenshot I posted was from the controller the manual page is on. So I was proposing using Master, since thats whats labelled on the controller.

Reply:
Ah I overlooked that. Okay, we can put it in a hwlabel to emphasize that it's the label on the device.